src/components/Report/Report.tsx

```diff
@@ -1,84 +1,162 @@
-import React from 'react'
-import './Report.css'
-import back_img from '../../images/report/Big Shoes - Jumping On One leg Pose.png'
-import location from '../../images/report/Charco - Location Map.png'
-import plant_1 from '../../images/extras/Fancy Plants - Solo Plant.png'
-import plant_2 from '../../images/extras/Fancy Plants - Solo Plant copy.png'
+//imports
+import React, {useContext, useState, useEffect} from 'react'
+import {ReportContext, AnswerContext} from '../../types'
+import { Cube } from '../Cube/Cube';
+import './Report.css';
+import { ReportCategory } from '../ReportCategory/ReportCategory';
+import {Spring} from 'react-spring/renderprops';
+import { Link } from 'react-router-dom'
+
+//assets
+import back_img from '../../images/report/Big Shoes - Jumping On One leg Pose.png';
+import location from '../../images/report/Charco - Location Map.png';
+import plan from '../../images/report/Charco - Good Job.png';
+import principal from '../../images/report/Charco - Inbox.png';
+import monthly from '../../images/report/Charco - Mobile Life.png';
+import downpayment from '../../images/report/Charco - Work at Home.png';
+
+
 
 const Report = () => {
+  
+  const categoryImages = [location, plan, principal, monthly, downpayment]
+  const [report, updateReport] = useState()
+  let userReport = useContext(ReportContext)
+  let answers = useContext(AnswerContext)
+
+  useEffect(()=>{
+    updateReport(userReport)
+  },[userReport])
+
+  const displayAnalysisSections = () =>{
+    const reportData = Object.keys(userReport)
+    
+    return reportData.map((data, key) =>{
+      let subtitle_1 = Object.keys(userReport[data])[0].replaceAll(/_|\-/g, " ")
+      let subtitle_2 = Object.keys(userReport[data])[1].replaceAll(/_|\-/g, " ")
+
+      return (
+        <ReportCategory 
+          
+          position={key + 1}
+          key={key}
+          plan={userReport[data].ten_year_plan}
+          categoryName={[data]}
+          categoryMainNumber={
+            userReport[data].zip_code
+            || userReport[data].based_on_rent
+            || userReport[data].monthly_principal
+            || userReport[data].down_payment_percentage_selected
+          }
+          categoryMainTitle={subtitle_1}
+          categorySubtitle={subtitle_2}
+          categorySecondNumber={
+            userReport[data].location
+            || userReport[data].goal_principal
+            || userReport[data].estimated_true_monthly
+            || userReport[data].down_payment_saved
```

Comment:
Wish there was a way we could clean this up, but I think this is the only way we can achieve this without adjusting our data model on the backend.